CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # `chainweb-node` Changelog
 
+## 2.17 (2022-11-17)
+
+This version replaces all previous versions. Any prior version will stop working
+on **2022-12-01T00:00:00Z**. Node administrators must upgrade to this version
+before that date.
+
+This version will expire on **2023-03-02T00:00:00Z**.
+
+To upgrade, pull the latest docker image or download the binary and restart the
+node.
+
+[Changes](https://github.com/kadena-io/chainweb-node/compare/2.16.1...2.17):
+
+*   Remove error messages from pact output for on-chain transactions. (#1543)
```

Comment:
Let's add a comment for builders: "Dapps can still retrieve transaction error messages from `local` endpoint."